<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>compare</class>
 <widget class="QDialog" name="compare">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>940</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>940</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Compare</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>302</width>
     <height>378</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelPoke1">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>270</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>270</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>302</width>
     <height>378</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QComboBox" name="comboBox_2">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelPoke2">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>270</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>270</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QChartView" name="Grafo">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
